src/test/scala/org/broadinstitute/dsde/vault/services/uBAMCollection/UBamCollectionDescribeServiceSpec.scala

```diff
@@ -0,0 +1,71 @@
+package org.broadinstitute.dsde.vault.services.uBAMCollection
+
+import org.broadinstitute.dsde.vault.model.uBAMCollectionJsonProtocol._
+import org.broadinstitute.dsde.vault.model.uBAMJsonProtocol._
+import org.broadinstitute.dsde.vault.model.{UBamCollection, UBamCollectionIngest, UBamIngest, UBamIngestResponse}
+import org.broadinstitute.dsde.vault.services.uBAM.UBamIngestService
+import org.broadinstitute.dsde.vault.{VaultConfig, VaultFreeSpec}
+import spray.http.StatusCodes._
+import spray.httpx.SprayJsonSupport._
+
+class UBamCollectionDescribeServiceSpec extends VaultFreeSpec with UBamCollectionDescribeService with UBamCollectionIngestService with UBamIngestService {
+
+  val routes = ubamCollectionDescribeRoute ~ uBamIngestRoute
+
+  def actorRefFactory = system
+
+  val versions = Table(
+    "version",
+    1
+  )
+
+  var testingUbamId = "ubam_invalid_UUID"
+  var testingubamCollectionId = "ubamCollection_invalid_UUID"
+
+
+  val files = Map(("bam", "vault/test/test.bam"), ("bai", "vault/test/test.bai"))
+  val metadata = Map("testAttr" -> "testValue")
+
+  val members = Option(Seq[String](testingUbamId))
+
+  "UBamCollectionDescribeServiceSpec" - {
+    forAll(versions) { (version: Int) =>
+
+      s"when accessing version = '${version}'" - {
+
+        "while preparing the ubamCollection test data" - {
+          "should successfully store the data using the UBam Ingest path" in {
+            val ubamIngest = new UBamIngest(files, metadata)
+            Post(VaultConfig.Vault.ubamIngestPath.versioned(version), ubamIngest) ~> addOpenAmCookie ~> uBamIngestRoute ~> check {
+              status should equal(OK)
+              testingUbamId = responseAs[UBamIngestResponse].id
+            }
+          }
+        }
+
+        "while preparing the  ubamCollection test data" - {
+          "should successfully store the data using the ubamCollection Ingest path" in {
+            val ubamCollectionIngest = new UBamCollectionIngest(members, metadata)
```

Comment:
`members` is initialized outside the loop, and hence is always `Option(Seq("ubam_invalid_UUID"))`.